crates/kitsune_p2p/crates/kitsune_p2p/tests/performance.rs

```diff
@@ -153,7 +154,7 @@ async fn minimise_p2p_agent_store_host_calls() {
         .filter(|e| matches!(e, RecordedKitsuneP2pEvent::QueryAgents { .. }))
         .count();
 
-    query_agents_count.assert_close_to(1400, 100);
+    query_agents_count.assert_close_to(1400, 200);
```

Comment:
I was hoping to keep this test working, but it looks like it's broken in other ways while not being run. I'll need to have a look separately at some point. Or maybe it just becomes irrelevant with the new peer store actually